### PR TITLE
Feature/use breakpoint

### PR DESCRIPTION
**Pull request by cloudratha**
- Adds Layout Component
- Adds SideBar and SideBarSection components
- Adds useBreakpoint and useBreakpointValue hooks
- Adds A11y tests for all components

### CODE REVIEW

File: src/components/center/center.test.tsx [hunk 1]
```diff
@@ -1,15 +1,17 @@
 import React from 'react';
 import { render } from '@testing-library/react';
 
+import { testA11y } from '../../test-utils';
 import { Center } from './center';
 
-test('it renders with both vertical and horizontal by default', () => {
+test('it renders with both vertical and horizontal by default', async () => {
```
Comment (RVerma90) [inline src/components/center/center.test.tsx:7]: Do we want to test for vertical  / horizontal cases separately too?

File: src/components/error-summary/error-summary.test.tsx [hunk 2]
```diff
@@ -1,11 +1,15 @@
...
 import { render, screen } from '@testing-library/react';
 
+import { testA11y } from '../../test-utils';
 import { ErrorSummary } from './error-summary';
 
-test('it renders an alert', () => {
-  render(<ErrorSummary id="error" title="Something went wrong" />);
+test('it renders an alert', async () => {
+  const { container } = render(
+    <ErrorSummary id="error" title="Something went wrong" />
```
Comment (RVerma90) [inline src/components/error-summary/error-summary.test.tsx:9]: Move "Something went wrong" to locale?

**Review by RVerma90 — APPROVED** [review]

File: src/components/error-summary/error-summary.test.tsx (hunk 2 above)
Comment (RVerma90) [reply src/components/error-summary/error-summary.test.tsx:9]: I think for tests it's fine actually

File: src/components/center/center.test.tsx (hunk 1 above)
Comment (cloudratha) [reply src/components/center/center.test.tsx:7]: there are tests for them individually, but if you mean test a11y its unnecessary as they just class applications.

**Pull request merged**